src/components/ProductForm/BackButton.js

```diff
@@ -0,0 +1,18 @@
+import { MdNavigateBefore } from 'react-icons/md'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+function BackButton() {
+  return (
+    <Button>
+      <MdNavigateBefore />
+      <Link to="/">Back to Products</Link>
+    </Button>
```

Comment:
Oh! You need to be careful here: a `button` is semantically different to an `anchor` element! You should either use a `button` or a `a` - not both!
```suggestion
    <>
      <MdNavigateBefore />
      <Link to="/">Back to Products</Link>
    </>
```
It is basically like this: 
<img width="459" alt="Bildschirmfoto 2021-09-22 um 11 05 15" src="https://user-images.githubusercontent.com/51355141/137328595-a561245f-6502-40e1-81f0-98f5ef20bc2b.png">

Reply:
Thanks, I appreciate the meme 😸 
Changed it to this 
```
<Link to="/">
      <MdNavigateBefore />
      Back to Products
    </Link>
```
.